backend-project/config/urls.py

```diff
@@ -53,6 +52,11 @@
     path("admin/", admin.site.urls),
     path("", include("small_eod.collections.urls")),
     path("", include("small_eod.cases.urls")),
+    #
+    path('letter/file/sign', PresignedUploadFileView.as_view(), name='file_upload'),
```

Comment:
Co z dokumentacją OpenAPI?

Reply:
Ups, to pochodzi z django-rest-frameworka, więc muszę się temu lepiej przyjrzeć i wprowadzić #178.